app/assets/javascripts/discourse/templates/user/bookmarks.hbs

```diff
@@ -0,0 +1,51 @@
+{{#if noContent}}
+  <div class='alert alert-info'>{{ noResultsHelp }}</div>
+{{else}}
+  {{#conditional-loading-spinner condition=loading}}
+  <table class="topic-list">
+    <thead>
+      <th>{{i18n "topic.title"}}</th>
+      <th>{{i18n "post.bookmarks.created"}}</th>
+      <th>{{i18n "activity"}}</th>
+      <th>&nbsp;</th>
+    </thead>
+    <tbody>
+    {{#each content as |bookmark|}}
+      <tr class="topic-list-item bookmark-list-item">
+        <td class="main-link">
+          <span class="link-top-line">
+            <div class="bookmark-metadata">
+              {{#if bookmark.bookmark_name}}
+                <span class="bookmark-metadata-item">
+                  {{d-icon "info-circle"}}{{bookmark.bookmark_name}}
+                </span>
+              {{/if}}
+              {{#if bookmark.bookmark_reminder_at}}
+                <span class="bookmark-metadata-item">
+                  {{d-icon "far-clock"}}{{bookmark.formattedReminder}}
+                </span>
+              {{/if}}
+            </div>
+
+            {{topic-status topic=bookmark}}
+            <a class="title" href="/t/{{bookmark.topic_id}}/{{bookmark.bookmark_post_number}}">{{bookmark.title}}</a>
```

Comment:
This is not going to work in subfolder installs.

Reply:
Fixed by just using topic-link helper instead :+1: